packages/fxa-content-server/tests/functional/post_verify/cad_qr.js

```diff
@@ -72,8 +73,20 @@ registerSuite('cad_qr_signin', {
           .then(
             testElementExists(selectors.POST_VERIFY_CAD_QR_GET_STARTED.HEADER)
           )
+          .then(click(selectors.POST_VERIFY_CAD_QR_GET_STARTED.LATER))
+          .getCurrentUrl()
+          .then(function (url) {
+            assert.isTrue(url.includes('mozilla.org'));
```

Comment:
Worth noting that I'm using a partial match check instead of `testUrlEquals(MOZ_ORG_SYNC_GET_STARTED_LINK)` because this address resolves to a locale-specific address (in my case `en-CA`) and thus always fails.